### PR TITLE
Component/table

### PR DESCRIPTION
**Pull request by iamayushy**
1. Normal
normal

2. Bordered
columnborder

3. Zebra
zebra

4. Condensed
condensed

5. Checkable
checked

6. Test
tests

7.Props
props

### CODE REVIEW

File: src/base/variable.less [hunk 1]
```diff
@@ -63,6 +63,8 @@
 
 
 @BaseFontSize: 10px;
+@FontMD: 16px;
```
Comment (ShahRohan27) [inline src/base/variable.less:66]: Not needed, use rem

File: src/components/NTable/NTable.vue [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+
+.n-table-footer {
+  padding: @SpacingM;
+  border-top: 1px solid @ColorPrimaryGrey40;
+  font-size: @rowfont;
+}
+
+.n-table-footer-condensed {
+  padding: @SpacingS;
+  border-top: 1px solid @ColorPrimaryGrey40;
```
Comment (ShahRohan27) [inline src/components/NTable/NTable.vue:319]: Please use rem everywhere

File: src/components/NTable/NTable.vue [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+  .n-table-row {
+    font-size: @rowfont - 2;
+    overflow: hidden;
+  }
+}
+
+.n-table-footer {
+  padding: @SpacingM;
+  border-top: 1px solid @ColorPrimaryGrey40;
+  font-size: @rowfont;
```
Comment (ShahRohan27) [inline src/components/NTable/NTable.vue:314]: use rem and try using the same casing for all variables

**Review by ShahRohan27 — CHANGES_REQUESTED** [review]

**Comment by ShahRohan27** [comment]
Also add on click row handlers.

**Comment by iamayushy** [comment]
@ShahRohan27 made the required changes. 
1. Changed from px to rem
2. Removed the variables
3. Added onRowClick event

**Comment by hiteshs01** [comment]
Merging as initial commit for table component

**Pull request merged**